### DISCUSSION TITLE
Heart ahoge <3 aw

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7765] [general]
![Screen Shot 2021-09-10 at 7 51 57 AM](https://user-images.githubusercontent.com/82217573/132856182-8afd69f1-f1b8-43ee-ab25-b165d5f75b45.png)

are there new ahoges? and if i wanna add new ones, can i justt copy and paste one of the jsons for the ahoges and edit it?

**Comment by briaryoung** [comment]
There are 10 random ahoges that you have a 50% chance of getting every morning. They are not spritepacks.

**Comment by briaryoung** [comment]
Here is the actual heart ahoge that I made for the game.

**Comment by briaryoung** [comment]
![678AE2CB-7376-4CA5-AE5F-24EFBEFD230C](https://user-images.githubusercontent.com/52429850/132857200-5e238c74-2dfd-4a32-a712-742bf8a96acc.jpeg)

**Comment by hlepnamingblyat** [comment]
How did you modified Monika's room?
How can I do that? I just can't find anything or any tutorial for it.

**Comment by ghost** [comment]
i got photos from the room selector submod, and i tok a room and renamed it
to spaceroom

**Comment by CrewieAustin** [comment]
Hm, I think I got a few of them before. From a straight ahoge to a square type ahoge, I think Heart Ahoge was epsecially super rare.

**Comment by AsuraShun** [comment]
What submods are you using?
